src/main/webapp/resources/js/pages/pipelines/pipeline.launch.js

```diff
@@ -7,11 +7,13 @@
 	 * @param ParameterService for passing parameter information between modal and page
 	 * @constructor
 	 */
-	function PipelineController($scope, $http, CartService, ParameterService) {
+	function PipelineController($scope, $http, CartService, ParameterService, ToolDataTableService) {
 		var vm = this;
 
 		vm.parameters = ParameterService.getOriginalSettings();
 		vm.selectedParameters = ParameterService.getSelectedParameters();
+		vm.toolDataTables = ToolDataTableService.getSettings();
+		vm.selectedToolDataTableField = ToolDataTableService.getSelectedToolDataTableField();
 
```

Comment:
If a pipeline does not require the use of a Galaxy Data Table, does the request require this information?

Reply:
I'm not sure exactly what you mean by 'request' here.

I've tried to set things up so that pipelines that don't pull data from a Galaxy Tool Data Table won't have any visible changes to their UI and will pass the same list of parameters to galaxy.

Are you suggesting eliminating `toolDataTables` and `selectedToolDataTableField` from the scope if the pipeline doesn't use a tool data table?